web/src/beta/features/PluginPlayground/Code/index.tsx

```diff
@@ -9,37 +16,54 @@ type Props = {
   executeCode: () => void;
 };
 
-const getLanguageByFileExtension = (fileTitle: string) => {
-  const ext = fileTitle.split(".").pop();
-  switch (ext) {
-    case "js":
-      return "javascript";
-    case "yml":
-    case "yaml":
-      return "yaml";
-    default:
-      return "plaintext";
-  }
-};
-
 const Code: FC<Props> = ({
   fileTitle,
   sourceCode,
   onChangeSourceCode,
   executeCode
 }) => {
+  const ediableHtmlSourceCode = useMemo(
```

Comment:
Sorry, I think there is a small typo here. Should it be `editableHtmlSourceCode` ?

Reply:
thanks!
I've fixed this commit
https://github.com/reearth/reearth-visualizer/pull/1236/commits/2e27ee6475a631c58c6b198c30ea89d66c16be7b